exampleSite/content/post/goodbye.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Goodby Friend"
```

Comment:
Is this file necessary here?

Reply:
Only added to confirm that the pagination was only working with posts. I can remove.